connectivity/builder/no_interrupted_connections.go

```diff
@@ -13,5 +15,15 @@ type noInterruptedConnections struct{}
 func (t noInterruptedConnections) build(ct *check.ConnectivityTest, _ map[string]string) {
 	newTest("no-interrupted-connections", ct).
 		WithCondition(func() bool { return ct.Params().IncludeConnDisruptTest }).
-		WithScenarios(tests.NoInterruptedConnections())
+		WithScenarios(tests.NoInterruptedConnections()).
+		WithFinalizer(func(ctx context.Context) error {
+			// Delete the test-conn-disrupt pods immediately after the test run
+			// has finished to reduce CPU consumption (the pods generate a lot
+			// of traffic, and in most CI jobs Cilium runs with monitor
+			// aggregation disabled).
+			if !ct.Params().ConnDisruptTestSetup {
+				return ct.DeleteConnDisruptTestDeployment(ctx, ct.K8sClient())
+			}
```

Comment:
The [old logic](https://github.com/cilium/cilium-cli/blob/pr%2Fbrb%2Fearly-terminate-conn-disrupt/connectivity/check/context.go#L423-L432) could be likely removed at this point. `DeleteConnDisruptTestDeployment` currently also prints a log line which could probably converted to debug level for consistency.

Reply:
Yep, I think it's safe to remove the old logic.